pom.xml

```diff
@@ -104,7 +104,91 @@
             </excludes>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-release-plugin</artifactId>
+          <version>2.5.3</version>
+          <configuration>
+            <mavenExecutorId>forked-path</mavenExecutorId>
+            <useReleaseProfile>false</useReleaseProfile>
+            <arguments>${arguments}</arguments>
```

Comment:
What CLI? I don't see them specified anywhere in the PR. Is the release plugin still used by Travis (or is this here for our Jenkins build)?